base/strings/search.jl

```diff
@@ -101,8 +101,24 @@ julia> findfirst("Julia", "JuliaLang")
 1:5
 ```
 """
-findfirst(pattern::AbstractString, string::AbstractString) =
-    findnext(pattern, string, firstindex(string))
+findfirst(pattern::AbstractString, string::AbstractString) = findnext(pattern, string, firstindex(string))
+
+"""
+    findfirst(ch::Char, string::AbstractString)
+
+Find the first occurrence of `ch` in `string`. Equivalent to
+[`findfirst(isequal(ch), string)`](@ref).
```

Comment:
As I noted in a comment, maybe use `==(ch)` instead? That sounds more natural.

Reply:
Okay, sure!

Reply:
Still present in the docstring.